src/with_clause_parser.h

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
I'd like to see testing of this module in particular through  regression tests driven C tests (like the net module is tested)